File: src/assets/Fleet.Assets/ViewModels/AssetViewModel.cs
using Fleet.Common;
using Fleet.Assets.Models;

namespace Fleet.Assets.ViewModels
{
    public class AssetViewModel
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public AssetCategory AssetCategory { get; set; }
        public Location LastKnownLocation { get; set; }
    }
}

File: src/ui/src/api/models/asset-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { AssetCategory } from './asset-category';
import { Location } from './location';
export interface AssetViewModel {
  assetCategory?: AssetCategory;
  id?: number;
  lastKnownLocation?: Location;
  name?: null | string;
}
